### ISSUE TITLE
Fedora rpminspect complains about the usage of inet_ntoa

### ISSUE THREAD

**Issue by mwperina** [engelmi/hirte#232]
Fedora rpminstpect complains, that using `inet_ntoa` function is forbidden:

```json
 "badfuncs": [
    {
      "result": "VERIFY",
      "waiver authorization": "Anyone",
      "message": "/usr/bin/hirte-agent may use forbidden functions on i686",
      "details": "Forbidden function symbols found:\n\tinet_ntoa\n",
      "remedy": "Forbidden symbols were found in an ELF file in the package.  The configuration settings for rpminspect indicate the named symbols are forbidden in packages.  If this is deliberate, you may want to disable the badfuncs inspection.  If it is not deliberate, check the man pages for the named symbols to see what API functions have replaced the forbidden symbols.  Usually a function is marked as deprecated but still provided in order to allow for backwards compatibility.  Whenever possible the deprecated functions should not be used."
    },
    {
      "result": "VERIFY",
      "waiver authorization": "Anyone",
      "message": "/usr/bin/hirte-agent may use forbidden functions on x86_64",
      "details": "Forbidden function symbols found:\n\tinet_ntoa\n",
      "remedy": "Forbidden symbols were found in an ELF file in the package.  The configuration settings for rpminspect indicate the named symbols are forbidden in packages.  If this is deliberate, you may want to disable the badfuncs inspection.  If it is not deliberate, check the man pages for the named symbols to see what API functions have replaced the forbidden symbols.  Usually a function is marked as deprecated but still provided in order to allow for backwards compatibility.  Whenever possible the deprecated functions should not be used."
    },
    {
      "result": "VERIFY",
      "waiver authorization": "Anyone",
      "message": "/usr/bin/hirte-agent may use forbidden functions on aarch64",
      "details": "Forbidden function symbols found:\n\tinet_ntoa\n",
      "remedy": "Forbidden symbols were found in an ELF file in the package.  The configuration settings for rpminspect indicate the named symbols are forbidden in packages.  If this is deliberate, you may want to disable the badfuncs inspection.  If it is not deliberate, check the man pages for the named symbols to see what API functions have replaced the forbidden symbols.  Usually a function is marked as deprecated but still provided in order to allow for backwards compatibility.  Whenever possible the deprecated functions should not be used."
    },
    {
      "result": "VERIFY",
      "waiver authorization": "Anyone",
      "message": "/usr/bin/hirte-agent may use forbidden functions on ppc64le",
      "details": "Forbidden function symbols found:\n\tinet_ntoa\n",
      "remedy": "Forbidden symbols were found in an ELF file in the package.  The configuration settings for rpminspect indicate the named symbols are forbidden in packages.  If this is deliberate, you may want to disable the badfuncs inspection.  If it is not deliberate, check the man pages for the named symbols to see what API functions have replaced the forbidden symbols.  Usually a function is marked as deprecated but still provided in order to allow for backwards compatibility.  Whenever possible the deprecated functions should not be used."
    },
    {
      "result": "VERIFY",
      "waiver authorization": "Anyone",
      "message": "/usr/bin/hirte-agent may use forbidden functions on s390x",
      "details": "Forbidden function symbols found:\n\tinet_ntoa\n",
      "remedy": "Forbidden symbols were found in an ELF file in the package.  The configuration settings for rpminspect indicate the named symbols are forbidden in packages.  If this is deliberate, you may want to disable the badfuncs inspection.  If it is not deliberate, check the man pages for the named symbols to see what API functions have replaced the forbidden symbols.  Usually a function is marked as deprecated but still provided in order to allow for backwards compatibility.  Whenever possible the deprecated functions should not be used."
    }
```

**Comment by engelmi** [comment]
Fixed in #233. We keep it open till we can verify the fix works in next release.

**Comment by engelmi** [comment]
This issue occurred in previous builds (see [here](https://bodhi.fedoraproject.org/updates/FEDORA-2023-97a6746095)), but is solved now (see [here](https://bodhi.fedoraproject.org/updates/FEDORA-2023-5180c96a66)).
